modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -94,6 +94,25 @@ enum class GShape: int
     GFRAME,
 };
 
+namespace gapi {
```

Comment:
Serialization stuff should be @ `s11n.hpp`, not here.

Reply:
Left only wrap_serialize here.